source/onboard/migrate-from-slack.rst

```diff
@@ -0,0 +1,191 @@
+
+  mmctl import upload ./mattermost-bulk-import.zip
+
+Run this command to list the available imports:
+
+.. code:: bash
+
+  mmctl import list available
+
+Run this command to process the import. Replace ``<IMPORT FILE NAME>`` with the name you got from the ``mmctl import list available`` command:
+
+.. code:: bash
+
+  mmctl import process <IMPORT FILE NAME>
+
+Finally, run this command to view the status of the import process job. If the job status shows as ``pending``, then wait before running the command again. The ``--json`` flag is required to view the possible error message. Replace ``<JOB ID>`` with the id you got from the ``mmctl import list process`` command:
+
+.. code:: bash
+
+  mmctl import job show <JOB ID> --json
+
+Option 2: Use the ``mattermost`` command to run the export
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First get the ``.jsonl`` file and ``data`` folder onto your server using something like ``scp``. Then you'll use the ``mattermost`` binary to process the bulk import data:
+
+.. code:: bash
+
+    mattermost import bulk ./mattermost_import.jsonl --import-path ./data --apply
+
+Debug imports
+^^^^^^^^^^^^^
+
+If you choose the ``mmctl`` option above, you can use the ``mmctl import job show`` command to view any relevant errors that may have occurred. If you run into problems which the error message does not help to resolve, please try the first option of using the ``mattermost bulk import`` command. The ``mmctl`` import process does not give you any additional debugging information, even in the Mattermost server logs.
+
+.. important::
+
+    Note that if your logged in Mattermost user is not present in your input (neither the email nor the username matches any users in the import file), your user will not be added to any new channels in the import process.
+
+Additional tools
+^^^^^^^^^^^^^^^^
+
+* `mm-emoji <https://github.com/maxwellpower/mm-emoji>`__ - Designed to smoothly transition emojis from a Slack instance to Mattermost.
+* `mm-importjs <https://github.com/mickmister/mmimportjs>`__ - Breaks up large import files into smaller ones, as well as automatically remove null characters in post content when importing data to Mattermost.
+* `slack-migrate-pinned-posts <https://github.com/svelle/slack-migrate-pinned-posts>`__ - Migrates pinned posts from Slack to Mattermost.
+
+Use the imported team
+^^^^^^^^^^^^^^^^^^^^^
+
+* During the import process, the emails and usernames from Slack are used to create new Mattermost accounts. If emails are not present in the Slack export archive, then placeholder values will be generated and the system admin will need to update these manually.
+* Slack users can activate their new Mattermost accounts by using Mattermost's **Password Reset** screen with their email addresses from Slack to set new passwords for their Mattermost accounts.
+* Once logged in, Mattermost users will have access to previous Slack messages in the public channels imported from Slack.
```

Comment:
I think a link that directs admins to instructyions for migrating users authentication method would be helpful.  I'm not sure if all the instructions should go here, or simply link off to our tool.  I'm inclined to put a full example for migrating email/pw users to SAML.